jwst/tweakreg/astrometric_utils.py

```diff
@@ -0,0 +1,179 @@
+import os
+import requests
+
+from astropy import table
+from astropy.table import Table
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+
+from resample import resample_utils
```

Comment:
Travis builds are failing on this. I think it needs to be "from ..resample import resample_utils" (i.e. a relative import)

Reply:
Fixed.